src/styles/variables.ts

```diff
@@ -152,6 +152,7 @@ export default {
     anonymousReportFooterBreakpoint: 650,
     dropDownButtonDividerHeight: 28,
     addPaymentMethodLeftSpacing: 2,
+    paymentMethodHeight: 64,
```

Comment:
@adhorodyski I just realized we already had a const for this `optionRowHeight` – in your next PR can you please get rid of this `paymentMethodHeight` and instead use `optionRowHeight`?

Reply:
Sure thing, will update in the morning! 🙂

Reply:
Please see my follow up PR https://github.com/Expensify/App/pull/30486 :)